tonami/Utterance.py

```diff
@@ -5,8 +5,8 @@
 from typing import Tuple, Union
 from pydub import AudioSegment
 
-import pitch_process as pp
-import audio_utils
+from tonami import pitch_process as pp
+from tonami import audio_utils
```

Comment:
assuming you tried this & it works, i think there was an issue or something with
"from tonami" when they were in the same folder

Reply:
yeah it seems like "from tonami" is required for anything outside of the tonami folder i.e. pytest and streamlit deployment 🤷‍♀️